Sources/XCHammer/main.swift

```diff
@@ -60,7 +60,7 @@ struct GenerateOptions: OptionsProtocol {
             forceRunOpt in { generateBazelTargetsOpt in { xcworkspacePathOpt -> GenerateOptions in
                 // Defaults to PWD
                 let workspaceRootPath: Path = workspaceRootPathOpt?.normalize() ??
-                    Path(FileManager.default.currentDirectoryPath)
```

Comment:
`required` revert. The expectation is that XCHammer uses `PWD` is the root of the workspace unless one is explicit set.